template.html

```diff
@@ -96,19 +96,19 @@ <h1>Glossar für Sprachmittler*innen im Kontext Flucht und Migration</h1>
 		<tfoot>
 			<tr>
 				<td class="fake-td">
-					<p>Das Glossar der KuB wird ständig überarbeitet und regelmäßig auf
-					<a href="https://kub-berlin.org/ueber-die-kub/veroeffentlichungen/">der Webseite</a>
-					veröffentlicht.</p>
-
 					<p>Die mit einem * markierten Begriffe haben wir mit freundlicher
 					Genehmigung der Schwulenberatung Berlin gGmbH aus ihrer
 					<a href="https://schwulenberatungberlin.de/post/handreichung-fuer-dolmetscher-innen/">Handreichung für Dolmetscher*innen</a> entnommen.</p>
-
-					<p>Es gibt auch ein <a href="https://www.bamf.de/SharedDocs/Anlagen/DE/EMN/Glossary/emn-glossary2.html">Glossar des Europäischen Migrationsnetzwerks (EMN)</a> mit vielen Wörtern und Übersetzungen in allen EU-Sprachen.</p>
 				</td>
 			</tr>
 		</tfoot>
 	</table>
+	<div>
+		<p>Das Glossar der KuB wird ständig überarbeitet und regelmäßig auf
+		<a href="https://kub-berlin.org/ueber-die-kub/veroeffentlichungen/">der Webseite</a>
+		veröffentlicht.</p>
+		<p>Es gibt auch ein <a href="https://www.bamf.de/SharedDocs/Anlagen/DE/EMN/Glossary/emn-glossary2.html">Glossar des Europäischen Migrationsnetzwerks (EMN)</a> mit vielen Wörtern und Übersetzungen in allen EU-Sprachen.</p>
+	</div>
```

Comment:
This way the styling for `tfoot` no longer applies. I suggest using a class (e.g. `notes`) on both `<tfoot>` and `<div>`

Reply:
> I suggest using a class (e.g. `notes`) on both `<tfoot>` and `<div>`

Like this?

Reply:
I wouldn't do the `padding-bottom: 0` (at least not this way) because it introduces a dependency between `.notes` and `.fake-td`